arm-apimanagement/2016-07-07/swagger/apimmanagement.json

```diff
@@ -4453,6 +4453,259 @@
           }
         }
       }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.ApiManagement/service/{serviceName}/backends": {
+      "get": {
+        "tags": [
+          "Backends"
+        ],
+        "operationId": "Backends_ListByService",
+        "description": "Lists a collection of backends in the specified service instance.",
+        "externalDocs": {
+          "url": "https://msdn.microsoft.com/en-us/library/dn935030.aspx"
+        },
+        "parameters": [
+          {
+            "$ref": "#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "$ref": "#/parameters/ServiceNameParameter"
+          },
+          {
+            "name": "$filter",
+            "in": "query",
+            "required": false,
+            "type": "string",
+            "description": "| Field | Supported operators    | Supported functions                         ||-------|------------------------|---------------------------------------------|| id    | ge, le, eq, ne, gt, lt | substringof, contains, startswith, endswith || host  | ge, le, eq, ne, gt, lt | substringof, contains, startswith, endswith |"
+          },
+          {
+            "name": "$top",
+            "in": "query",
+            "required": false,
+            "type": "integer",
+            "format": "int32",
+            "description": "Number of records to return."
+          },
```

Comment:
#fixed

---
In reply to: [88086762](https://github.com/Azure/azure-rest-api-specs/pull/705#discussion_r88086762) [](ancestors = 88086762)